app/assets/javascripts/steps.js

```diff
@@ -15,6 +15,7 @@
       this.bindStatusClicks();
       this.bindCancelAddChangeNoteLink();
       this.bindCancelAddSecondaryLink();
+      this.bindOverviewTableFilter();
```

Comment:
Can you add some detail to the commit note. For example what does this method have to do with using frontend components? Why "on change" being used now rather than "on click"?

Reply:
https://github.com/alphagov/govuk_admin_template/blob/master/spec/javascripts/spec/filterable_table.spec.js